### PR TITLE
chore: update heartbeat interval logic

### PR DESCRIPTION
**Pull request by makeavish**
#### Screenshots

NA

### CODE REVIEW

File: pkg/query-service/telemetry/telemetry.go [hunk 1]
```diff
@@ -213,8 +206,11 @@ func createTelemetry() {
 
 	telemetry.SetTelemetryEnabled(constants.IsTelemetryEnabled())
 
-	ticker := time.NewTicker(HEART_BEAT_DURATION)
-	activeUserTicker := time.NewTicker(ACTIVE_USER_DURATION)
+	// Create a new scheduler
+	s := gocron.NewScheduler(time.UTC)
```
Comment (prashant-shahi) [inline pkg/query-service/telemetry/telemetry.go:210]: Context usage doesn't look right.

Please refer to [this](https://github.com/SigNoz/saas-scripts/blob/main/sqlite-backup/cron.go#L14).

File: pkg/query-service/telemetry/telemetry.go [hunk 2]
```diff
@@ -227,141 +223,171 @@ func createTelemetry() {
...
+		spansInLastHeartBeatInterval, _ := telemetry.reader.GetSpansInLastHeartBeatInterval(context.Background(), HEART_BEAT_DURATION)
+		getSamplesInfoInLastHeartBeatInterval, _ := telemetry.reader.GetSamplesInfoInLastHeartBeatInterval(context.Background(), HEART_BEAT_DURATION)
+		totalSamples, _ := telemetry.reader.GetTotalSamples(context.Background())
+		tsInfo, _ := telemetry.reader.GetTimeSeriesInfo(context.Background())
+
+		getLogsInfoInLastHeartBeatInterval, _ := telemetry.reader.GetLogsInfoInLastHeartBeatInterval(context.Background(), HEART_BEAT_DURATION)
+
+		traceTTL, _ := telemetry.reader.GetTTL(context.Background(), &model.GetTTLParams{Type: constants.TraceTTL})
+		metricsTTL, _ := telemetry.reader.GetTTL(context.Background(), &model.GetTTLParams{Type: constants.MetricsTTL})
+		logsTTL, _ := telemetry.reader.GetTTL(context.Background(), &model.GetTTLParams{Type: constants.LogsTTL})
```
Comment (prashant-shahi) [inline pkg/query-service/telemetry/telemetry.go:260]: Context usage doesn't look right.

We should create single context and pass it here.

File: pkg/query-service/telemetry/telemetry.go (hunk 1 above)
Comment (makeavish) [reply pkg/query-service/telemetry/telemetry.go:210]: I am using V1 and the above reference uses v2.

File: pkg/query-service/telemetry/telemetry.go (hunk 2 above)
Comment (makeavish) [reply pkg/query-service/telemetry/telemetry.go]: Done

**Review by prashant-shahi — APPROVED** [review]

**Pull request merged**